pkg/sidecar/config/config.go

```diff
@@ -296,6 +301,16 @@ func (s *ScyllaConfig) validateCpuSet(ctx context.Context, cpusAllowed string, s
 	return nil
 }
 
+func setupIOTuneCache() error {
+	if _, err := os.Stat(scyllaIOPropertiesPath); errors.Is(err, os.ErrNotExist) {
```

Comment:
nit: would you mind splitting it on 2 lines for better readability? (also the other)

Reply:
splitted, left second if using init-statement becasue it looks natural